### ISSUE TITLE
Tooltip's HTML <select> is not clickable

### ISSUE THREAD

**Issue by Morgs007** [highcharts/highcharts#6312]
#### Expected behaviour
In a tooltip with HTML enabled tag with dropdown/select to open with available dropdown options.

#### Actual behaviour
The dropdown/select doesn't open, seems disabled?

**Comment by pawelfus** [comment]
Hi @Morgs007 

It's caused by fix for #295 (demo for firefox: http://jsfiddle.net/Mscva/3/ ). 

**Workaround:**
Overwrite `onContainerMousedown` and disable selection in browsers by CSS:

```javascript
Highcharts.Pointer.prototype.onContainerMouseDown = function(e) {
      e = this.normalize(e);
      this.zoomOption(e);
      this.dragStart(e);
}
```

```css
.highcharts-container {
      -webkit-touch-callout: none;
      -webkit-user-select: none;
      -khtml-user-select: none;
      -moz-user-select: none;
      -ms-user-select: none;
      user-select: none;
}
```

**Comment by Morgs007** [comment]
Thanks @pawelfus, got it to work and had to comment out 
`//this.zoomOption(e); `
as it was turning out undefined...
